src/index.ts

```diff
@@ -182,6 +182,10 @@ function publishStatement (context: SplitExecutionContext): void {
 }
 
 function handleKeyTokenFindResult (context: SplitExecutionContext, findResult: FindExpResult): void {
+  // ignore case of delimiter command
+  if (findResult.exp?.trim().toUpperCase() === "DELIMITER") {
+    findResult.exp = "DELIMITER";
+  }
   switch (findResult.exp?.trim()) {
```

Comment:
I would rather implement in this way to avoid unnecessary `if` statement branch and variable assignment. What do you think?
```suggestion
  switch (findResult.exp?.trim().toUpperCase()) {
```